contrib/util/reset-ps-all-apps.sh

```diff
@@ -0,0 +1,12 @@
+apps=$(curl -H "Authorization: token $DEIS_TOKEN" http://$DEIS_SERVER/v1/apps  | jq -r '.results | map(.id) | join(" ")')
```

Comment:
Yes except for the pesky header it includes. I know I could get rid of that with a simple grep -v but I needed to use jq anyways